stdlib/3/re.pyi

```diff
@@ -71,6 +71,10 @@ else:
     TEMPLATE = 0
     _FlagsType = int
 
+if sys.version_info <= (3, 6):
```

Comment:
This must be `< (3, 7)`, this is also why flake8 complains. Reason:

```python
>>> (3, 6, 1) <= (3, 6)
False
```

Reply:
Gotcha. Thanks Sebastian!